CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration.uc

```diff
@@ -570,6 +571,19 @@ static protected function EventListenerReturn AfterActionModifyRecoveredLoot (Ob
 	return ELR_NoInterrupt;
 }
 
+static protected function EventListenerReturn WillRecoveryTimeModifier(Object EventData, Object EventSource, XComGameState GameState, Name Event, Object CallbackData)
+{
+	local XComLWTuple Tuple;
+	
+	Tuple = XComLWTuple(EventData);
+
+	if (Tuple == none || Tuple.Id != 'WillRecoveryTimeModifier') return ELR_NoInterrupt;
+
+	Tuple.Data[0].f = 1.0 - class'X2Helper_Infiltration'.static.GetRecoveryTimeModifier();
```

Comment:
Could you integrate `1.0 -` into `GetRecoveryTimeModifier()`? Really confusing to read this line 😄